### ISSUE TITLE
Merge Problem

### ISSUE THREAD

**Issue by florian33** [vehiclefits/vfmagento#54]
Hi,

We are expecting some troubles.
I'm using the merge module.
What's that currently doing, is deleting all the Vehicule Application.
I don't know if it's only my case.
I create a F-250 and F-250 Light Duty.
F-250 Light Duty 1990-2013 is applied for a product. 
I want to merge both of them in F-250.
What I finally had is F-250 1999 for the product, others are lost.

Someone get the same trouble ?

**Comment by florian33** [comment]
I tried with a fresh installation, here is what I've done : 
1 - I create 1 make with 2 models and 4 years for bothh of them
Mazda Mazdaspeed 3 1980 to 1983
Mazda Ram 3 2030 to 2033

2 - I create a simple Product and assign all my vehicle
![image](https://f.cloud.github.com/assets/666505/1095775/2859ae66-16f0-11e3-9ffd-0be1e968adf8.png)

3 - Search in the frontend works well. So i try to merge Mazdaspeed 3 into Ram 3
![image](https://f.cloud.github.com/assets/666505/1095785/7e353404-16f0-11e3-9ef7-a4909c781037.png)

4- it keeps years in Vehicle manage section
![image](https://f.cloud.github.com/assets/666505/1095792/ba917ca0-16f0-11e3-8aea-4bdfffc666dc.png)

5- But doesn't keep Vehicule Application : 
![image](https://f.cloud.github.com/assets/666505/1095800/dd6e9ef6-16f0-11e3-8489-044024b9cdc4.png)

Hope that will help you !

**Comment by florian33** [comment]
I tried this on test website : 
Error 504 Gateway timeout and same problem
You can check it with SKu #Wizzotest1 on test website

**Comment by joshribakoff** [comment]
The first post, step 3 looks like the correct behaviour, it merges the models but leaves the years as is. Instead I suppose you'd like it to have all the years from both vehicles? That actually makes sense, I guess I coded it in a confusing way. The second issue just shows that the operation took longer than your php.ini allowed.

A good workaround is to export your CSV, and edit the fitments there then re-import. But if you can elaborate on how the behavior should change, I can look into changing it. FYI a future version will have a faster merge operation, so the timeout is less of an issue. For now you can increase php.ini settings

**Comment by florian33** [comment]
Yes, exactly , in step 4 we would like to see Ram 3 1980 - 1983 and Ram 3 2030-2033 if that's possible.

Thanks you.
